test/spec/startup_script_stdlib_version_spec.bats

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Google LLC. This software is provided as-is,
+# without warranty or representation for any use or purpose.
+# Your use of it is subject to your agreements with Google.
+
+load "test_helper/load"
+
+@test "STARTUP_SCRIPT_STDLIB_INITIALIZED is initialized to 0" {
+  assert_equal "${STARTUP_SCRIPT_STDLIB_INITIALIZED}" "0"
+}
+
+# These error codes are expected to be part of a stable API
+@test "E_RUN_OR_DIE error code is 5" {
+  assert_equal "${E_RUN_OR_DIE}" "5"
+}
+
+@test "E_MISSING_MANDATORY_ARG error code is 9" {
+  assert_equal "${E_MISSING_MANDATORY_ARG}" "9"
+}
+
+@test "E_UNKNOWN_ARG error code is 10" {
+  assert_equal "${E_UNKNOWN_ARG}" "10"
+}
```

Comment:
Review note: https://github.com/terraform-google-modules/terraform-google-startup-scripts/pull/4 expands test coverage; in the mean time this is satisfactory to prove out the rest of testing.